src/Mentat.UI/Views/Shared/_Layout.cshtml

```diff
@@ -6,6 +6,8 @@
     <title>@ViewData["Title"] - Mentat</title>
     <link rel="stylesheet" href="~/css/fsu-theme.css" />
     <link rel="stylesheet" href="~/css/site.css" />
+
```

Comment:
whitespace- but it may be intentional separating our stylesheets from third-parties. Idk best practice here.